### PR TITLE
Pixel projectiles! Oh BOY!

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Ported over from Atlas with all fixes included.
Also includes cluster grenades. They're adminspawn-only atm.

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -1549,7 +1549,6 @@
 #include "code\modules\space transition\space_transition.dm"
 #include "code\modules\surgery\cavity_implant.dm"
 #include "code\modules\surgery\core_removal.dm"
-#include "code\modules\surgery\dental_surgeries.dm"
```
Comment (Carbonhell) [inline tgstation.dme:1552]: ???

File: tgstation.dme [hunk 2]
```diff
@@ -1089,7 +1090,6 @@
 #include "code\modules\food&drinks\recipes\tablecraft\recipes_spaghetti.dm"
 #include "code\modules\games\cards.dm"
 #include "code\modules\holiday\easter.dm"
-#include "code\modules\holiday\halloween.dm"
```
Comment (Carbonhell) [inline tgstation.dme:1092]: wot

File: tgstation.dme (hunk 1 above)
Comment (AliiTea) [inline tgstation.dme:1552]: Fix these

File: tgstation.dme (hunk 2 above)
Comment (AliiTea) [inline tgstation.dme:1092]: Fix these

**Comment by Crystalwarrior** [comment]
Fixed

**Comment by danhibiki** [comment]
Sure, why not

**Comment by TDcoolguy300** [comment]
I hope you tested this

**Comment by Crystalwarrior** [comment]
I did test it, didn't find any problems.

**Pull request merged**